gluster_exporter/volinfo.go

```diff
@@ -0,0 +1,233 @@
+package main
+
+import (
+	"encoding/json"
+	"encoding/xml"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+var (
+	glusterMgmt                = "glusterd2"
+	glusterd2Endpoint          = "http://localhost:24007"
+	glusterCmdGd1              = "gluster"
+	gd1Volinfo                 []Volume
+	gd2Volinfo                 []Volume
+	volinfoRefreshIntervalMins float64 = 15
+	invalidTime                        = time.Date(1970, 1, 1, 0, 0, 0, 0, time.UTC)
+	gd1VolinfoCacheTime                = invalidTime
+	gd2VolinfoCacheTime                = invalidTime
+)
+
+type Brick struct {
```

Comment:
Lets move the definitions to some model package so that it can be used in other places as well.

Reply:
ack

Reply:
I suggested @sidharthanup  to create `glusterutils` package for the same. I was planning to move this to there(https://github.com/gluster/gluster-prometheus/pull/38)

We need to converge on the package name which provides the gluster related functionalities.